Tests/SourceKitLSPTests/SemanticTokensTests.swift

```diff
@@ -905,7 +905,7 @@ final class SemanticTokensTests: XCTestCase {
       expected: [
         TokenSpec(marker: "1️⃣", length: 4, kind: .keyword),
         TokenSpec(marker: "2️⃣", length: 3, kind: .identifier),
-        TokenSpec(marker: "3️⃣", length: 3, kind: .function),
+        TokenSpec(marker: "3️⃣", length: 3, kind: .function, modifiers: .argumentLabel),
         TokenSpec(marker: "4️⃣", length: 3, kind: .struct, modifiers: .defaultLibrary),
         TokenSpec(marker: "5️⃣", length: 3, kind: .function),
         TokenSpec(marker: "6️⃣", length: 3, kind: .function),
```

Comment:
Shouldn’t this token also have a `.argumentLabel` modifier? 

Also in case you see the test as being skipped on your machine, you need to set `SOURCEKIT_TOOLCHAIN_PATH` to a Swift Development Snapshot as described here: https://github.com/swiftlang/sourcekit-lsp/blob/main/CONTRIBUTING.md#selecting-a-toolchain